src/view/droppable-dimension-publisher/droppable-dimension-publisher.jsx

```diff
@@ -23,6 +23,10 @@ export default class DroppableDimensionPublisher extends Component {
   /* eslint-disable react/sort-comp */
   props: Props;
 
+  static defaultProps = {
```

Comment:
because this is an internal component we do not need to use default props - we can do the defaulting in the public `Droppable`